tests/testthat/test-cpp11.R

```diff
@@ -5,12 +5,14 @@ test_that("use_cpp11() requires a package", {
 })
 
 test_that("use_cpp11() creates files/dirs, edits DESCRIPTION and .gitignore", {
+  skip_if_not_installed("cpp11")
+
   create_local_package()
   use_roxygen_md()
+  use_package_doc()
 
   local_interactive(FALSE)
   local_check_installed()
-  local_mocked_bindings(check_cpp_register_deps = function() invisible())
 
```

Comment:
In terms of skips and mocks, I would expect this to look very much like the same test for `use_rcpp()`:

https://github.com/r-lib/usethis/blob/a91943706708f7ab41105cc3d5edf6ee61bbfb15/tests/testthat/test-rcpp.R#L6-L21

Reply:
I changed it to reflect this. The test "use_rcpp_armadillo() creates Makevars..." is something I can implement later for cpp11armadillo.